lms/templates/main.html

```diff
@@ -48,6 +48,8 @@
       </title>
   </%block>
 
+  <script type="text/javascript" src="${static.url("lms/js/iframe-render.js")}"></script>
```

Comment:
@VladyslavTy, maybe we should consider adding the `defer` option? It allows the script to load in parallel and execute only after the DOM tree has been rendered.